packages/ember-data/lib/system/relationships/belongs-to.js

```diff
@@ -102,6 +102,14 @@ function belongsTo(modelName, options) {
 
   return computedPolyfill({
     get: function(key) {
+      if (opts.hasOwnProperty('serialize')) {
```

Comment:
Can you put the `opts.hasOwnProperty('serialize')` inside the warn, so it will get stripped in production?